install_config/aggregate_logging.adoc

```diff
@@ -462,252 +525,299 @@ $ for dc in $(oc get deploymentconfig --selector logging-infra=elasticsearch -o
+----
+
+. After the intended number of deployment configurations are created, scale up
+each new one to deploy it:
 +
+----
+$ oc scale --replicas=1 dc/logging-es-<suffix>
+----
+
+=== Fluentd
+
+Once Elasticsearch is running, label nodes to enable Fluentd to run on them
+and feed logs to Elasticsearch. Use the `*fluentd-nodeselector*` given to
+the deployer (if different) in the command below:
+
 ====
 ----
-$ oc edit template logging-es[-ops]-template
+$ oc label nodes --all logging-infra-fluentd=true
 ----
 ====
-+
-. In addition to updating the template, all of the deployment configurations for
-that cluster also need to have the three environment variable values above
-updated. To edit each of the configurations for the cluster in series, you use
-the following.
-+
+
+=== Kibana
+
+To access the Kibana console from the {product-title} web console, add the
+`loggingPublicURL` parameter in the *_/etc/origin/master/master-config.yaml_*
+file, with the URL of the Kibana console (the `*kibana-hostname*` parameter).
+The value must be an HTTPS URL:
+
 ====
 ----
-$ oc get dc -l component=es[-ops] -o name | xargs -r oc edit
+...
+assetConfig:
+  ...
+  loggingPublicURL: "https://kibana.example.com"
+...
 ----
 ====
-+
-. Create an additional deployment configuration, run the following
-command against the Elasticsearch cluster you want to to scale up for
-(*logging-es-template* or *logging-es-ops-template*).
-+
+
+Setting the `loggingPublicURL` parameter creates a *View Archive* button on the
+{product-title} web console under the *Browse* -> *Pods* -> *<pod_name>* ->
+*Logs* tab. This links to the Kibana console.
+
+You can scale the Kibana deployment as usual for redundancy:
+
 ====
 ----
-$ oc new-app logging-es[-ops]-template
+$ oc scale dc/logging-kibana --replicas=2
 ----
 ====
-+
-These deployments will be named differently, but all will have the *logging-es*
-prefix. Be aware of the cluster parameters (described in the deployer
-parameters) based on cluster size that may need corresponding adjustment in the
-template, as well as existing deployments.
 
-. After the intended number of deployment configurations are created, scale up
-your cluster, starting with Elasticsearch as described in
-xref:../install_config/upgrading/manual_upgrades.adoc#manual-upgrading-efk-logging-stack[Upgrading
-the EFK Logging Stack].
-+
-[NOTE]
+You can see the UI by visiting the site specified at the `*KIBANA_HOSTNAME*`
+variable.
+
+See the https://www.elastic.co/guide/en/kibana/4.1/discover.html[Kibana
+documentation] for more information on Kibana.
+
+[[configuring-curator]]
+=== Curator
+
+Curator allows administrators to configure scheduled Elasticsearch maintenance
+operations to be performed automatically on a per-project basis. It is scheduled
+to perform actions daily based on its configuration. Only one Curator pod is
+recommended per Elasticsearch cluster. Curator is configured via a mounted YAML
```

Comment:
well it's a configmap that *contains* a file...

since there's a whole section about how to get the file in there, seems simplest just to omit mention of mounting here.